flake.nix

```diff
@@ -7,14 +7,30 @@
 
   outputs = { self, nixpkgs, flake-utils, ocaml-overlays }:
     flake-utils.lib.eachDefaultSystem (system:
-      let pkgs = import nixpkgs { inherit system; overlays = [ocaml-overlays.overlay]; };
-      in
-      {
+      let
+        pkgs = import nixpkgs {
+          inherit system;
+          overlays = [ ocaml-overlays.overlay ];
+        };
+      in {
         devShell = (pkgs.mkShell {
           shellHook = ''
             npm install
             export PATH="node_modules/.bin:_build/default/src/bin:$PATH"
           '';
+          buildInputs = with pkgs; [
+            # Make develop life easier
+            # General tooling
+            docker
+            docker-compose
+
+            # OCaml developer tooling
+            opam
+            pkgs.nodePackages.esy
```

Comment:
We probably want to add @d4hines `esy-fhs` for greater compatibility

Reply:
What is that?

Reply:
He has a flake that constructs a environment that works for `esy` here: https://github.com/d4hines/esy-fhs

Reply:
I think it's used in `main` so we can just re-add it here

Reply:
Maybe, I don't have any idea on "how to add it" would you mind git it a try?

Reply:
Done :)

Reply:
Took the freedom to remove opam as well since it's not used in Deku